build.gradle

```diff
@@ -1103,8 +1103,14 @@ project(':clients') {
   }
 
   compileJava.dependsOn 'processMessages'
+  compileJava {
+    options.compilerArgs << "-Xlint:rawtypes"
+  }
 
   compileTestJava.dependsOn 'processTestMessages'
+  compileTestJava {
+    options.compilerArgs << "-Xlint:rawtypes"
+  }
```

Comment:
Would it be better to keep this in the central place where we set the javac configs, but check the project name? It will be easier to maintain that way, I think.